examples/material/create_material.py

```diff
@@ -1,37 +1,44 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+# <a href="https://colab.research.google.com/github/Exabyte-io/exabyte-api-examples/blob/feature/SOF-4400-skinny-req/examples/material/create_material.ipynb" target="_blank">Open in Google Colab</a>
+
 # # Overview
 # 
 # In this example we create a material from a JSON config with [tags](https://docs.exabyte.io/entities-general/data/#tags) to identify the material.
 
-# # Execution
+# # Complete Authorization Form and Initialize Settings
 # 
-# > <span style="color: orange">**NOTE**</span>: In order to run this example, an active Exabyte.io account is required. RESTful API credentials shall be updated in [settings](../settings.py). The generation of the credentials is also explained therein.
+# This will also determine environment and set all environment variables. We determine if we are using Jupyter Notebooks or Google Colab to run this tutorial. NOTE: If you are running this notebook from Jupyter, the variables ACCOUNT_ID, AUTH_TOKEN, MATERIALS_PROJECT_API_KEY, and ORGANIZATION_ID should be set in the file 'settings.json' if you need to use these variables.
```

Comment:
Let's include everything after "NOTE" as a quote similar to: https://github.com/Exabyte-io/exabyte-api-examples/blob/feature/SOF-4400-skinny-req/examples/material/get_materials_by_formula.ipynb now
![Screen Shot 2021-06-30 at 12 58 54 PM](https://user-images.githubusercontent.com/721112/124023584-eed81500-d9a2-11eb-892a-c927cd3995ca.png)

Reply:
A link to "settings.json" could be useful here too, just like in the visual above